e2e/e2e_test.go

```diff
@@ -345,7 +364,7 @@ func TestE2E(t *testing.T) {
 			verify: []verifyCmd{
 				{
 					from: "sleep.foo", to: "httpbin.foo:8000", path: "/headers",
-					want: []string{"By=spiffe://ymzhu-istio.svc.id.goog/ns/foo/sa/httpbin", "URI=spiffe://ymzhu-istio.svc.id.goog/ns/foo/sa/sleep"},
+					want: []string{newIdentity("By=","foo", "httpbin"), newIdentity("URI=","foo", "sleep")},
```

Comment:
there should be a space between `"By=","foo"`, similar format issues here and some other places, you could just format the whole file with the command `go fmt e2e/e2e_test.go`